app/gui/view/component-browser/component-list-panel/breadcrumbs/src/lib.rs

```diff
@@ -128,7 +128,7 @@ struct Layers {
 impl Layers {
     /// Constructor.
     pub fn new(base_layer: &Layer) -> Self {
-        let mask = Layer::new_with_cam("mask", &base_layer.camera());
+        let mask = Layer::new_with_cam_parent("mask", base_layer);
```

Comment:
This should rather be named `new_with_parent_cam` to sound more like a correct English phrase